BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -427,3 +405,71 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     }
   }
 }
+
+extension WalletViewController {
+  
+  func startNotificationObserver() {
+    // Stopping as a precaution
+    // Add observer
+    NotificationCenter.default.addObserver(self, selector: #selector(notificationAdded(_:)), name: NSNotification.Name.BATBraveLedgerNotificationAdded, object: nil)
+    loadNextNotification()
+  }
+  
+  @objc private func notificationAdded(_ notification: Notification) {
+    // TODO: Filter notification?
+    // LoadNext if there is no current notification
+    if currentNotification == nil {
+      loadNextNotification()
+    }
+  }
+  
+  private func clearCurrentNotification() {
+    if let currentNotification = currentNotification {
+      state.ledger.clearNotification(currentNotification)
+      self.currentNotification = nil
+    }
+  }
+  
+  private func loadNextNotification() {
+    //.failedContribution, .adsLaunch, .invalid & .backupWallet are not supported right now
+    // TODO: Verify that notifications are ordered & they are displayed in FIFO.
+    let ignoredRewardNotificationTypes: [RewardsNotification.Kind] = [.failedContribution,
```

Comment:
We have this list of ignored notifications in 2 places now, is it possible to consolidate it somehow?

Reply:
I can only see this. Can you mark the other place?

Reply:
https://github.com/brave/brave-rewards-ios/pull/123/files#diff-1ed66972ddc2dd1522f79ac2731e9a61R17

Implicitly here

Reply:
well that list is used to filter unwanted builder calls, and first check to stop handling certain notifications. Builder can still have the code to create view for unwanted notifications. Removed filter for now